WORKSPACE

```diff
@@ -16,8 +16,8 @@ workspace(name = "build_bazel_rules_typescript")
 
 git_repository(
     name = "build_bazel_rules_nodejs",
-    remote = "https://github.com/bazelbuild/rules_nodejs",
-    tag = "0.1.11",
+    commit = "f6eb823e1140da6eafbf77f56a249f72bb89416e",
+    remote = "https://github.com/achew22/rules_nodejs",
```

Comment:
do you have a separate PR to get that change upstream?

Reply:
Its already merged :)